src/ts/components/prototyping/dab-ipsum.tsx

```diff
@@ -34,49 +34,28 @@ const ipsumItem = (component: DabIpsumProps['component'], index: number) => {
 /**
  * Custom Ipsum component, useful for rendering placeholder text when prototyping.
  */
-export class DabIpsum extends Component<DabIpsumProps, {}> {
-  public shouldComponentUpdate(prevProps: DabIpsumProps) {
-    return (
-      prevProps.component !== this.props.component ||
-      prevProps.count !== this.props.count
-    );
-  }
-
-  public render() {
-    const { component = 'p', count = 5 } = this.props;
+const DabIpsum = (props: DabIpsumProps) => {
+  const { component = 'p', count = 5 } = props;
 
-    const items = Array.apply(null, new Array(count));
+  const items = [...Array(count)];
 
-    switch (component) {
-      case 'ul':
-        return (
-          <ul>
-            {items.map((value: void, index: number) =>
-              ipsumItem(component, index)
-            )}
-          </ul>
-        );
-      case 'ol':
-        return (
-          <ol>
-            {items.map((value: void, index: number) =>
-              ipsumItem(component, index)
-            )}
-          </ol>
-        );
-      case 'text':
-        return ipsumItem(component, 0);
-      // case 'p'
-      default:
-        return (
-          <div>
-            {items.map((value: void, index: number) =>
-              ipsumItem(component, index)
-            )}
-          </div>
-        );
-    }
+  switch (component) {
+    case 'ul':
+      return (
+        <ul>{items.map((_value, index) => ipsumItem(component, index))}</ul>
+      );
+    case 'ol':
+      return (
+        <ol>{items.map((_value, index) => ipsumItem(component, index))}</ol>
+      );
+    case 'text':
+      return ipsumItem(component, 0);
+    // case 'p'
```

Comment:
Think you wanna handle or remove this :)

Reply:
Nah, this is to let peeps know that the `p` case will fall through to the `default`. I could just uncomment it, but the code will run exactly the same.

Reply:
I reckon uncomment is probably clearer that its intentional and handled so would get my vote :)